Jenkinsfile

```diff
@@ -66,11 +66,19 @@ pipeline {
     }
     stage('Lint'){
       options { skipDefaultCheckout() }
+      when {
+        beforeAgent true
+        expression { return env.ONLY_DOCS == "false" }
+      }
```

Comment:
I'm not sure we should include this guard. 

As an example, let's say someone committed changes to https://github.com/elastic/beats/blob/master/filebeat/module/apache/_meta/docs.asciidoc but forgot to run `mage update` and commit the results of it as well. This linting stage here in CI that runs `make check` would catch such an issue. So we'd want it to run even when only documentation files are changed in a PR.

Reply:
I was in fact working on where to enable this `mage update` command, if the linting is the one responsable for this particular analysis independently of the module, then it will definitely simplify the approach I was tempted to move forward :)